src/settings/App.tsx

```diff
@@ -161,6 +177,34 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 		return parseInt(OC.config.version,10) >= 16;
 	}
 
+	showAdminDelegationForms() {
+		if (this.state.isAdminNextcloud && this.state.checkAppsInstalled) {
+			return <div id="groupfolders-admin-delegation">
+				<h3>{ t('groupfolders', 'Group folder admin delegation') }</h3>
+				<em>{ t('groupfolders', 'Nextcloud allows you to delegate the administration of groupfolders to non-admin users.') }</em>
+				<br/>
+				<br/>
+				<em>{ t('groupfolders', 'Specify below the groups that will be allowed to manage groupfolders and use its API/REST.') }</em>
+				<br/>
+				<em>{ t('groupfolders', 'They will have access to all Groupfolders.') }</em>
+				<AdminGroupSelect
+					groups={this.state.groups}
+					allGroups={this.state.groups}
+					delegatedAdminGroups={this.state.delegatedAdminGroups} />
+				<br/>
+				<em>{ t('groupfolders', 'Specify below the groups that will be allowed to manage groupfolders and use its API/REST only.') }</em>
+				<br/>
+				<em>{ t('groupfolders', 'They will only have access to Groupfolders for which they have advanced permissions.') }</em>
+				<SubAdminGroupSelect
+					groups={this.state.groups}
+					allGroups={this.state.groups}
+					delegatedSubAdminGroups={this.state.delegatedSubAdminGroups} />
+				<br/>
+				<br/>
```

Comment:
Say no to `<br>` :) also I wonder if we really need `<em>` everywhere

```suggestion
				<p><em>{ t('groupfolders', 'Nextcloud allows you to delegate the administration of groupfolders to non-admin users.') }</em><p>
				<p><em>{ t('groupfolders', 'Specify below the groups that will be allowed to manage groupfolders and use its API/REST.') }</em></p>
				<p><em>{ t('groupfolders', 'They will have access to all Groupfolders.') }</em></p>
				<AdminGroupSelect
					groups={this.state.groups}
					allGroups={this.state.groups}
					delegatedAdminGroups={this.state.delegatedAdminGroups} />
				<p><em>{ t('groupfolders', 'Specify below the groups that will be allowed to manage groupfolders and use its API/REST only.') }</em></p>
				<p><em>{ t('groupfolders', 'They will only have access to Groupfolders for which they have advanced permissions.') }</em></p>
				<SubAdminGroupSelect
					groups={this.state.groups}
					allGroups={this.state.groups}
					delegatedSubAdminGroups={this.state.delegatedSubAdminGroups} />
```

Reply:
I don't think it's very necessary to use `<em>` here ?

Reply:
By the way, why do I say no to `<br>` ? ^^'